src/customerio-cdp.ts

```diff
@@ -18,32 +17,27 @@ import { assert, validate } from './utils/param-validation';
 
 const packageJson = require('customerio-reactnative/package.json');
 
-// Ensures all methods defined in codegen spec are implemented by the public module
-interface NativeSpec extends Omit<CodegenSpec, keyof TurboModule> {
```

Comment:
Remove for now due to static vs instance method conflicts. We can reimplement it with public api improvements if needed.